### ISSUE TITLE
Rules for zen mode nav

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#2989]
Zen mode nav was created, I believe, for two purposes:
- petition pages: so users would be focused on signing the petition 
- mini-site pages: initiatives would feel like they had their own dedicated website

Now that we are trying to increase traffic across our site and trying to link our work together more holistically, I think we should reduce our use of Zen mode to only petition pages. @xmatthewx @beccaklam what do you think? 

If we set up some rules then we can talk to devs about removing the checkbox in the cms tab and make templates zen mode or regular automatically.

**Comment by kristinashu** [comment]
Note: we decided the blog detail pages should use Zen Mode for a more focused reading experience and we will use Related Posts at the bottom of the page to keep visitors on our site.

**Comment by Pomax** [comment]
I ran into this today while browsing the blog, noticing that on blog pages the nav was suddenly gone, and initially thought that was a bug. I think if it's content on our site, consistently showing the nav, with the nav housing all the things we want people to be able to instantly hop over to (so probably add `blog`, too, for example?) is better than having some pages have a nav, and others not.

**Comment by cadecairos** [comment]
@kristinashu is this work still relevant or should we close it?

**Comment by kristinashu** [comment]
Let's keep this open!

@xmatthewx @taisdesouzalessa @beccaklam do you have any insights or recommendations for this?

**Comment by taisdesouzalessa** [comment]
I agree with @kristinashu  in keeping the Zen mode to a minimum (like only in petitions) so the site is easier to navigate by always having the top elements visible. 

Even in the campaigns I question if it really makes a lot of difference having them there at the top, but I need some data to back up this guess.

**Comment by beccaklam** [comment]
I also think we should keep Zen mode use to a minimum, if at all. I think having the navigation constantly there is helpful to the user and not a distraction, even on petitions. I also need to back this up with data, but my gut says people will check out what's on the page first, and only if what they found interests them will they even bother to look at the nav. cc: @kristinashu @taisdesouzalessa @xmatthewx

**Comment by xmatthewx** [comment]
I think I agree all around. We should show the nav by default just about everywhere, probably even on blog post pages (though maybe we wait for revised nav to land for that). We can remove or hide that zen nav option from other templates. We shouldn't get rid of it totally, so we can quickly deliver special snowflakes to put out sudden fires of imagination should they arise.

**Comment by kristinashu** [comment]
@Pomax do you have any thoughts or concerns on this? Recommendations are:
- Remove zen mode nav from all (with the exception of YouTube Regrets) our templates 
- Keep zen mode on YouTube Regrets template
- Make it _easy_ to use zen mode on future special projects like YouTube Regrets
- But wait until design is done with our current IA work before make these implementations

**Comment by Pomax** [comment]
the youtube regrets page is a special kind of page that has its own template, so even there I would opine we don't need a zen mode at the CMS level, but we can "define" the page as not having the full top nav by making sure the template doesn't use it. (In that sense, it's also easy to ensure future special projects don't have a full nav.)

**Comment by xmatthewx** [comment]
Another thought: we could hide the author option, but leave the underlying functionality? Wait and see if comes up before we fully deprecate it?
